test/src/test/java/org/corfudb/infrastructure/log/StreamLogFilesTest.java

```diff
@@ -736,4 +744,34 @@ public void estimateSizeTest() throws IOException {
         assertThat(parentSize).isEqualTo(parentDirFilePayloadSize + childDirFilePayloadSize);
         assertThat(childDirSize).isEqualTo(childDirFilePayloadSize);
     }
+
+    @Test
+    public void readConsistencyTest() throws Exception {
```

Comment:
Even if `fsync` is not executed yet it doesn't mean that data is corrupted. 
In very rare cases data can get corrupted on disk and we can't do anything with that, there is no point to bring additional latency and wait until fsync happens - it's a pretty expensive operation.
Also, these changes can affect reads, which can cause unexpected behavior.